src/ui/composer.ui

```diff
@@ -96,7 +96,7 @@
           </sizepolicy>
          </property>
          <property name="toolTip">
-          <string>&lt;p&gt;Save (&lt;span style=&quot; color:#565656;&quot;&gt;Shift+Tab&lt;/span&gt;)&lt;/p&gt;</string>
+          <string>&lt;p&gt;Save (&lt;span style=" color:#565656;"&gt;Shift+Tab&lt;/span&gt;)&lt;/p&gt;</string>
```

Comment:
This is okay, and XML parsers do handle it. Mudlets own XML serialiser does it this way. As a bonus, it's a bit more readable!

🤔 it might even look better Crowdin for translators...

Reply:
I'll be honest - I'd have expected this or the previous point to have changed the content such that it breaks something somehow... if it really doesn't than I can't protest *that* much...

Reply:
`"` are valid in text node of XML tag. The would need to be escaped or encoded in attribute.

Reply:
:+1: Okay then. I don't have any objections now.